libs/template/renderer.go

```diff
@@ -160,7 +131,26 @@ func (r *renderer) computeFile(relPathTemplate string) (*inMemoryFile, error) {
 	}
 	perm := info.Mode().Perm()
 
+	// If file name does not specify the `.tmpl` extension, then it is copied
+	// over as is, without treating it as a template
+	if !strings.HasSuffix(relPathTemplate, templateExtension) {
+		return &copyFile{
+			fileCommon: &fileCommon{
+				root:    r.instanceRoot,
+				relPath: relPathTemplate,
+				perm:    perm,
+			},
```

Comment:
This can be constructed above and reused below to reduce duplication.

Reply:
Relative path is different for them, so we can't. Keeping it as is.